src/ClientInterface.php

```diff
@@ -155,7 +155,7 @@
  * @method mixed             mset(array $dictionary)
  * @method int               msetnx(array $dictionary)
  * @method Status            psetex(string $key, $milliseconds, $value)
- * @method Status            set(string $key, $value, $expireResolution = null, $expireTTL = null, $flag = null)
+ * @method Status|null       set(string $key, $value, $expireResolution = null, $expireTTL = null, $flag = null)
```

Comment:
Can you add a test case for this?

Reply:
Happy to add a test. Can you point me into the right direction? Are you looking for a test in `ClientTest` or are there integrations tests that interact with Redis?

Reply:
I'd suggest adding a new method to `SET_Test.php` and doing the `$this->assetNull()` to ensure that's the case.

Reply:
It appears there already is a test that asserts that calling `SET` with `NX` will return `null` if the key is set already: `testSetStringValueWithModifierNX`

Reply:
Perfect.